charts/monitoring/grafana/dashboards/minio/overview.json

```diff
@@ -232,12 +232,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "rate(minio_network_sent_bytes_total[$interval])",
+          "expr": "rate(s3_tx_bytes_total{app=\"minio\"}[$interval])",
```

Comment:
for me something does not seem right with this part of the change. What I mean by that is: yes it is fixing our dashboard, but `s3_tx...` and `s3_rx..` are legacy metrics that are deprecated and should not be used anymore. `network_sent_bytes_total` and `network_received_bytes_total` are the current ones. So we are somehow moving from the current metrics to the old ones, which does not seem right to me 🤔 


This can be quickly verified by doing

```bash
export MINIO_PROMETHEUS_AUTH_TYPE=public
./minio server /tmp/data


# using the current endpoint, will yield no result
curl -s http://192.168.0.61:9000/minio/v2/metrics/cluster | grep rx 

# using the legacy endpoint, will result in the metric
❯ curl -s http://192.168.0.61:9000/minio/prometheus/metrics | grep rx
internode_rx_bytes_total 0
s3_rx_bytes_total 0
```

I am also a bit confused about the old change as from the looks of it our scraping path has not been changed in the past 2 years https://github.com/kubermatic/kubermatic/blob/98a76b89094ed3f53c68ba87485259577f6914b3/charts/minio/templates/deployment.yaml#L32
Has the original ever worked?

All in all this PR is fine in the sense that it does fix the dashboard, but I am wondering if we have a deeper issue in how we scrape minio, that has impact on this change.

Reply:
@SimonTheLeg yes, you are right. As mentioned in my [previous comment](https://github.com/kubermatic/kubermatic/pull/8687#issuecomment-1012025667), I also discovered that we use the "legacy" metrics endpoint of MinIO.

However we are not moving from the current metrics to the old ones: If you take a look at the [migration guide](https://github.com/Praveenrajmani/minio/blob/9ff7af2dfbc14d971fd3ce8fd02ae32b4252e6d4/docs/metrics/prometheus/README.md#migration-guide-for-the-new-set-of-metrics) the metric `minio_network_sent_bytes_total` was indeed renamed to `s3_tx_bytes_total` (or split into `s3_tx_bytes_total` and `internode_tx_bytes_total`) and is not part of the currently [exposed metrics](https://github.com/minio/minio/blob/RELEASE.2022-01-08T03-11-54Z/docs/metrics/prometheus/list.md).

I agree with you that we should use the current metrics endpoints (`minio/v2/metrics/cluster` and `minio/v2/metrics/node`). I don't have a strong opinion when we want to implement this change: I can take a look at the metrics exposed by the v2 metrics endpoints and implement the necessary changes right away or I can also create a follow-up issue to discuss this in more depth. What do you think?